### DISCUSSION TITLE
Adding the same attributes to a list of weapons

### DISCUSSION THREAD

**Discussion by higps** [nosoop/SM-TFCustAttr#16] [Q&A] [answered]
Do I have to copy paste attributes to the same weapon, or is there a syntax to apply the same attributes to different weapons?

    // Buff Banner
    "129"
    {
        "banner rage fill multiplier"            "1.2"
    }

    // Festive Buff Banner
    "1001"
    {
        "banner rage fill multiplier"            "1.2"
    }
	
	
	in to
	
	    // Buff Banner
    "129 1001"
    {
        "banner rage fill multiplier"            "1.2"
    }

I tried with various separators, but none seemed to work. Is this supported or do I just need to copy paste each weapon instance?

**Comment by nosoop** [comment, marked as answer]
The manager is intended to be a simple reference implementation for other authors to modify and adapt to their own plugins; separators are not supported.
